tests/commons/j2/test_j2_functions.py

```
@@ -11,290 +12,250 @@
+
+@pytest.mark.parametrize("case", [
+    (("Hello!", "\"Hello!\"^^xsd:string", '"')),
+    (("Hello!", "'Hello!'^^xsd:string")),
+    (("'", "'\\''^^xsd:string")),
+    (("'", '"\'"^^xsd:string', '"')),
+    (('"', "'\"'^^xsd:string")),
+    (('"', '"\\""^^xsd:string', '"')),
+    ((">'<", "'>\\'<'^^xsd:string")),
+    ((">\n<", "'''>\n<'''^^xsd:string")),
+    ((">\n<", '""">\n<"""^^xsd:string', '"')),
+    (("Hello!", "'Hello!'^^xsd:string")),
```

Ruff (0.8.2)

Duplicate of test case at index 1 in `pytest.mark.parametrize`

Remove duplicate test case

(PT014)